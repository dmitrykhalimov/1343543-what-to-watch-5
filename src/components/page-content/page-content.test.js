import React from "react";
import renderer from "react-test-renderer";
import PageContent from "./page-content";

const MockComponent = () => {

  return (
    <div>
    </div>
  );
};

describe(`PageContent`, () => {
  it(`Should PageContent render correctly`, () => {
    const tree = renderer
      .create(
          <PageContent>
            <MockComponent/>
          </PageContent>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
