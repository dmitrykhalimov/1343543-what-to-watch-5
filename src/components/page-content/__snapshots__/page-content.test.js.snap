// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent Should PageContent render correctly 1`] = `
<div
  className="page-content"
>
  <div />
</div>
`;
